src/widget/woverview.cpp

```diff
@@ -661,6 +843,143 @@ void WOverview::drawCurrentPosition(QPainter* pPainter) {
+        }
+        pPainter->setPen(shadowPen);
+        pPainter->drawLine(line);
+
+        pPainter->setPen(Qt::green);
+        pPainter->drawLine(line);
+
+        QPointF textPoint = m_timeRulerPos;
+        QPointF textPointDistance = m_timeRulerPos;
+        double widgetPositionFraction;
+        double padding = 1.0; // spacing between line and text
+        if (m_orientation == Qt::Horizontal) {
+            textPoint = QPointF(textPoint.x() + padding, fontMetrics.height());
+            textPointDistance = QPointF(textPointDistance.x() + padding, height() / 2);
+            widgetPositionFraction = m_timeRulerPos.x() / width();
+        } else {
+            textPoint.setX(0);
+            textPointDistance.setX(0);
+            widgetPositionFraction = m_timeRulerPos.y() / height();
+        }
+        double trackSamples = m_trackSamplesControl->get();
+        double timePosition = samplePositionToSeconds(
+                widgetPositionFraction * trackSamples);
+        double timePositionTillEnd = samplePositionToSeconds(
+                (1 - widgetPositionFraction) * trackSamples);
+        double timeDistance = samplePositionToSeconds(
+                (widgetPositionFraction - m_playpositionControl->get()) * trackSamples);
+
+        QString timeText = mixxx::Duration::formatTime(timePosition)
+                + " -" + mixxx::Duration::formatTime(timePositionTillEnd);
+
+        m_timeRulerPositionLabel.prerender(textPoint, QPixmap(), timeText,
+                markerFont, Qt::white, m_labelBackgroundColor, width(), getDevicePixelRatioF(this));
+        m_timeRulerPositionLabel.draw(pPainter);
+
+        QString timeDistanceText;
+        if (timeDistance > 0) {
+            timeDistanceText = mixxx::Duration::formatTime(timeDistance);
+        }
+        m_timeRulerDistanceLabel.prerender(textPointDistance, QPixmap(), timeDistanceText,
+                markerFont, Qt::white, m_labelBackgroundColor, width(), getDevicePixelRatioF(this));
+        m_timeRulerDistanceLabel.draw(pPainter);
+    } else {
+        m_timeRulerPositionLabel.clear();
+        m_timeRulerDistanceLabel.clear();
+    }
+}
+
+void WOverview::drawMarkLabels(QPainter* pPainter, const float offset, const float gain) {
+    QFont markerFont = pPainter->font();
+    markerFont.setPixelSize(10 * m_scaleFactor);
+    QFontMetricsF fontMetrics(markerFont);
+
+    QFont shadowFont = pPainter->font();
+    shadowFont.setWeight(99);
+    shadowFont.setPixelSize(10 * m_scaleFactor);
+
+    // Draw WaveformMark labels
+    for (const auto& pMark : m_marksToRender) {
+        if (m_pHoveredMark != nullptr && pMark != m_pHoveredMark) {
+            if (pMark->m_label.intersects(m_pHoveredMark->m_label)) {
+              continue;
+            }
+        }
+        if (pMark->m_label.intersects(m_cuePositionLabel)
+            || pMark->m_label.intersects(m_cueTimeDistanceLabel)
```

Comment:
4 space more indent. I think clang wants tailing operators, but I don't care.

Reply:
I hesitate with this because it can make it hard to see where the `if` condition ends and the block inside it begins. How to work around that? Add a blank line after the `{`?

Reply:
4 space more indent are the important part.  The extra indent makes it clear because the block content has less indent.
 
I think git-clang-format will unwap the lines if the operators are in front ... unsure.

Reply:
Oh sorry, I misunderstood. Yes, an extra 4 spaces makes it clear where the block starts.